rasa/core/tracker_store.py

```diff
@@ -250,8 +250,8 @@ def deserialise_tracker(
         try:
             dialogue = Dialogue.from_parameters(json.loads(serialised_tracker))
         except UnicodeDecodeError as e:
-            raise ValueError(
-                "Tracker can not be deserialised. "
+            raise RasaException(
```

Comment:
how about a more specific subtype of `RasaException`?

Reply:
I will create a new one, unless you have one in mind?

Reply:
had a look but couldn't find something similar. Let's create a new one 👍🏻